microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -92,8 +92,12 @@ See xref:../microshift_running_apps/microshift-gitops.adoc#microshift-gitops[Aut
 ==== Getting a cluster ID
 With this release, you can get the ID of a {microshift-short} cluster. When opening a support case, you can provide the cluster ID to Red{nbsp}Hat Support to help in identifying issues with your cluster. See xref:../microshift_support/microshift-getting-cluster-id.adoc#microshift-getting-cluster-id[Getting your cluster ID] for more information.
 
-//[id="microshift-4-16-security"]
-//=== Security and compliance
+[id="microshift-4-16-security"]
+=== Security and compliance
+
+[id="microshift-4-16-ssl-medium-cipher-suites"]
+==== SSL Medium Strength Cipher Suites now supported
+During an SSL handshake between a client and a server, the cipher to use is negotiated between them. With this release, SSL Medium Strength Cipher Suites are now supported for the kube-controller-manager daemon, kube-scheduler control-plane process, and kubelet "node agent." This enhancement to the internal communication between kubernetes components improves control plane communications security.
```

Comment:
Do we want to reference the issue? https://issues.redhat.com/browse/OCPBUGS-29037